system/HTTP/CURLRequest.php

```diff
@@ -705,9 +707,16 @@ protected function setCURLOptions(array $curlOptions = [], array $config = [])
         }
 
         // Cookie
-        if (isset($config['cookie'])) {
-            $curlOptions[CURLOPT_COOKIEJAR]  = $config['cookie'];
-            $curlOptions[CURLOPT_COOKIEFILE] = $config['cookie'];
+        if ($cookie = ($config['cookie_jar'] ?? $config['cookie'] ?? false)) {
+            $curlOptions[CURLOPT_COOKIEJAR] = $cookie;
+        }
+
+        if ($cookie = ($config['cookie_file'] ?? $config['cookie'] ?? false)) {
+            $curlOptions[CURLOPT_COOKIEFILE] = $cookie;
+        }
```

Comment:
I think this would be cleaner if we didn't reuse variables:
```suggestion
        if ($cookieJar = ($config['cookie_jar'] ?? $config['cookie'] ?? false)) {
            $curlOptions[CURLOPT_COOKIEJAR] = $cookieJar;
        }

        if ($cookieFile = ($config['cookie_file'] ?? $config['cookie'] ?? false)) {
            $curlOptions[CURLOPT_COOKIEFILE] = $cookieFile;
        }
```